pkg/controller/servingcert/starter/starter.go

```diff
@@ -31,6 +37,11 @@ func StartServiceServingCertSigner(ctx *controllercmd.ControllerContext) error {
 		return err
 	}
 
+	intermediateCACert, err := readIntermediateCACert(config.IntermediateCertFile)
```

Comment:
nit: would prefer to have the `len == 0` logic here. Noop funcs with empty parameters are always suspicious.

Reply:
I'm afraid I'm not sure I understand what you mean by `Noop funcs with empty parameters`.  I've added a comment indicating that the intermediate cert will only be expected after rotation, does that help?

Reply:
would check for the filename to be non-empty here, not inside of a func (which has no clue about the context it is called in).

Reply:
Done